src/cpp/session/modules/SessionDataImportV2.R

```diff
@@ -13,6 +13,27 @@
 #
 #
 
+.rs.addFunction("path_relative_to_working_dir", function(target)
+{
+   if (is.null(target))
+   {
+      target
+   }
+   else
+   {
+      currentPath <- paste(getwd(), .Platform$file.sep, sep = "")
```

Comment:
Ah, interesting -- I was mistaken then! (A bit surprising since the default for e.g. `normalizePath()` does uses `\` slashes on Windows)

Let's leave it as `/` then; sorry about that!